### PR TITLE
update taquito docs to florence

### LINKED ISSUE #822: Update Taquito website to use Florencenet
The live code examples are using edonet. We need to update the website to use florencenet. 
For live code examples that make calls to a smart contract, we need to deploy a similar contract on Florencenet and replace the address in the examples.

Deploying all the contracts used in the examples is quite time-consuming.
It is a good occasion to find how we can make this task more efficient.

### PR DESCRIPTION
**Pull request by michaelkernaghan**
CLOSE #822 

- [x] Your code builds cleanly without any errors or warnings

Document for how to originate new contracts for the examples nad for tracking the changes:

https://docs.google.com/document/d/15rfuo70GkMHiKV_ChrRUbLcxOUo7S6hegQ4pzX21WlQ/edit?usp=sharing

### CODE REVIEW

File: integration-tests/contract-fetch-multiple-big-map-keys.spec.ts [hunk 1]
```diff
@@ -31,7 +31,7 @@ CONFIGS().forEach(({ lib, rpc, setup }) => {
                 }
             });
             const contract = await op.contract();
-
+            console.log("originates a contract with empty bigmap and fetches the storage/bigmap contract address : "+contract.address)
```
Comment (roxaneletourneau) [inline integration-tests/contract-fetch-multiple-big-map-keys.spec.ts:34]: We can remove this console.log

File: integration-tests/lambda-view.spec.ts [hunk 2]
```diff
@@ -47,6 +47,7 @@ CONFIGS().forEach(({ lib, rpc, setup }) => {
 
       await op.confirmation()
       const contract = await op.contract();
+      console.log("Originate FA1.2 contract and fetch data from view entrypoints contract address : "+contract.address)
```
Comment (roxaneletourneau) [inline integration-tests/lambda-view.spec.ts:50]: same as above

File: integration-tests/lambda-view.spec.ts [hunk 3]
```diff
@@ -93,7 +94,7 @@ CONFIGS().forEach(({ lib, rpc, setup }) => {
 
       await op.confirmation()
       const contract = await op.contract();
-
+      console.log("Originate a contract and fetch data from view entrypoints contract address : "+contract.address)
```
Comment (roxaneletourneau) [inline integration-tests/lambda-view.spec.ts:97]: same as above

File: integration-tests/lambda-view.spec.ts [hunk 4]
```diff
@@ -146,6 +147,7 @@ CONFIGS().forEach(({ lib, rpc, setup }) => {
 
       await op.confirmation()
       const contract = await op.contract();
+      console.log("Originate FA2 contract and fetch data from view entrypoints contract address : "+contract.address)
```
Comment (roxaneletourneau) [inline integration-tests/lambda-view.spec.ts:150]: same as above

File: docs/michelsonmap.md [hunk 5]
```diff
@@ -15,7 +15,7 @@ Taquito reads maps in the storage of smart contracts and translates them into an
 - _The key/value methods_: they allow you to manipulate the keys and values in the map
 - _The update methods_: they transform the map itself, for example, by deleting elements or clearing out the map entirely.
 
-This tutorial uses a [simple smart contract deployed on Edonet](https://better-call.dev/edo2net/KT1E7jtyoejiFWJnomA51SSW6jtZnwTApHT8/operations) with a map that contains addresses as keys and tez as values. We will use all the methods available in Taquito's `MichelsonMap` to check the map, extract values and modify them!
+This tutorial uses a [simple smart contract deployed on florencenet](https://better-call.dev/florencenet/KT1E7jtyoejiFWJnomA51SSW6jtZnwTApHT8/operations) with a map that contains addresses as keys and tez as values. We will use all the methods available in Taquito's `MichelsonMap` to check the map, extract values and modify them!
 
 If you want to see all these functions in action, you can have a look at [this Replit repo](https://repl.it/@claudebarde/TaquitoMichelsonMap#index.ts) and click on the "Run" button.
```
[the remark this answers is not in the document]
Comment (michaelkernaghan) [reply docs/michelsonmap.md:20]: yeah, i wondered about that. good thanks, will remove it.

[the remark this answers is not in the document]
Comment (michaelkernaghan) [reply docs/michelsonmap.md]: change done

File: integration-tests/lambda-view.spec.ts (hunk 2 above)
Comment (michaelkernaghan) [reply integration-tests/lambda-view.spec.ts:50]: ok./ i thought they might be useful for next time we want to re originate for a new protocol - I will remove them

File: integration-tests/contract-fetch-multiple-big-map-keys.spec.ts (hunk 1 above)
Comment (michaelkernaghan) [reply integration-tests/contract-fetch-multiple-big-map-keys.spec.ts:34]: removed.

File: docs/quick_start.md [hunk 6]
```diff
@@ -160,10 +160,10 @@ Tezos.contract
...
   })
-  .then((hash) => println(`Operation injected: https://edo.tzstats.com/${hash}`))
+  .then((hash) => println(`Operation injected: https://florence.tzstats.com/${hash}`))
   .catch((error) => println(`Error: ${JSON.stringify(error, null, 2)}`));
 ```
 
 [boilerplate]: https://github.com/ecadlabs/taquito-boilerplate
 [smart_contract_source]: https://ide.ligolang.org/p/CelcoaDRK5mLFDmr5rSWug
-[smart_contract_on_better_call_dev]: https://better-call.dev/edo2net/KT1F7DYSa7fVTNScSDDVVokqmmytpJBB5bs9/operations
+[smart_contract_on_better_call_dev]: https://better-call.dev/florencenet/KT1F7DYSa7fVTNScSDDVVokqmmytpJBB5bs9/operations
```
Comment (kjaiswal) [inline docs/quick_start.md:169]: I couldn't find this contract on BCD. Screenshot attached - 
Screen Shot 2021-05-03 at 9 01 11 AM

**Review by kjaiswal — CHANGES_REQUESTED** [review]

File: docs/quick_start.md (hunk 6 above)
Comment (michaelkernaghan) [reply docs/quick_start.md:169]: its there for me:

https://better-call.dev/search?text=KT1F7DYSa7fVTNScSDDVVokqmmytpJBB5bs9

Comment (michaelkernaghan) [reply docs/quick_start.md]: thanks, the link that shows where the contact can be found on BCD is a link to the wrong contract. I have corrected it.

**Review by roxaneletourneau — APPROVED** [review]

**Pull request merged**